app/_src/quickstart/universal-demo.md

```diff
@@ -125,148 +131,140 @@ kuma-dp run \
-By default, a very permissive traffic permission is created.
+This command will create a policy that allows all traffic between applications within your mesh. If you need to create 
+more specific rules, you can do so by editing the policy manifest.
+{% endif_version %}
 
-For the sake of this demo, we will delete it:
+We can enable Mutual TLS with a `builtin` CA backend by executing:
 
-```sh
-kumactl delete traffic-permission allow-all-default
+```shell
+echo 'type: Mesh
+name: default
+mtls:
+  enabledBackend: ca-1
+  backends:
+    - name: ca-1
+      type: builtin' | kumactl apply -f -
 ```
 
-You can try to make requests to the demo application at [`127.0.0.1:5000/`](http://127.0.0.1:5000/) and you will notice that they will **not** work.
+The traffic is now encrypted with mTLS and each service can reach any other service.
 
-Now, let's add back the default traffic permission:
-```sh
-echo 'type: MeshTrafficPermission
-name: allow-all
-mesh: default
-spec:
-  targetRef:
-    kind: Mesh
-  from:
-  - targetRef:
-      kind: Mesh
-    default:
-      action: Allow' | kumactl apply -f -
+We can then restrict the traffic by default by executing:
+
+```shell
+echo 'type: MeshTrafficPermission 
+name: mtp
+mesh: default 
+spec: 
+  targetRef: 
+    kind: Mesh 
+  from: 
+    - targetRef: 
+        kind: Mesh 
+      default: 
+        action: Deny' | kumactl apply -f -
 ```
 
-By doing so every request, we now make sure our demo application at [`127.0.0.1:5000/`](http://127.0.0.1:5000/) is not only working again, but it's automatically encrypted and secure.
+At this point, the demo application should not function, because we blocked the traffic.
+You can verify this by clicking the increment button again and seeing the error message in the browser.
+We can allow the traffic from the `demo-app` to `redis` by applying the following `MeshTrafficPermission`:
+
+```shell
+echo 'type: MeshTrafficPermission 
+name: allow-from-demo-app
+mesh: default 
+spec: 
+  targetRef: 
+    kind: MeshSubset
+    tags:
+      kuma.io/service: redis
+  from: 
+    - targetRef: 
+        kind: MeshSubset 
+        tags:
+          kuma.io/service: demo-app
+      default: 
+        action: Allow' | kumactl apply -f -
+```
 
-{% tip %}
-As usual, you can visualize the Mutual TLS configuration and the Traffic Permission policies we have just applied via the GUI, the HTTP API or `kumactl`.
-{% endtip %}
+You can click the increment button, the application should function once again.
+However, the traffic to `redis` from any other service than `demo-app` is not allowed.
 
 ## Next steps
 
 * Explore the [Features](/features) available to govern and orchestrate your service traffic.
-* Add a gateway to access the demo from the outside by following the [builtin gateway guide](/docs/{{ page.version }}/guides/gateway-builtin/).
```

Comment:
removed this links as they are Kubernetes guides, and userdon't have setup to follow them after this guide